test/other-issues/sqlite-relations-too-many-parameters/sqlite-relations-too-many-parameters.ts

```diff
@@ -51,9 +51,8 @@ describe("other issues > sqlite relations too many parameters", () => {
         const postWithCategory = await connection.manager.findOne(Post, 1, {
             relations: ["categories"]
         });
-        console.log(postWithCategory);
 
-        postWithCategory!.categories.length.should.be.equal(2000);
+        postWithCategory!.categories.length.should.be.equal(1200);
```

Comment:
I have no idea why it was 2000 before, we've got just 1200 items.

Reply:
@Kononnable I have some not pushed changes on my machine, let me push them before merging these changes. Please take a look on them

Reply:
weird but test is passing on master? How it is?

Reply:
Test was introduced in next, it doesn't exist in master. It was introduced during time, when only 20 tests were run(`.only` problem), after removing `.only` tests on CI always failed(and bailed) before that test.